packages/account/src/Components/poi-unsupported-failed/unsupported-failed.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import IconMessageContent from 'Components/icon-message-content';
 
-const UnsupportedFailed = ({ error }) => (
+const UnsupportedFailed = ({ error }: { error: string }) => (
```

Comment:
could you please use here separate type definition

Reply:
Since it was just one prop, I used it in the same line